superbuild/CMakeLists.txt

```diff
@@ -124,6 +131,28 @@ if (EXTERNAL_VERBOSE)
     set(EXTERNAL_VERBOSE_FLAG "-v")
 endif()
 
+if (JOBS)
+    set(NINJA_JOBS "-j ${JOBS}")
+elseif($ENV{JOBS})
+    set(NINJA_JOBS "-j $ENV{JOBS}")
+else()
+    set(NINJA_JOBS "")
+endif()
+
+if (BUILD_TYPE)
+    set(BUILD_TYPE_ISPC ${BUILD_TYPE})
+    set(BUILD_TYPE_LLVM ${BUILD_TYPE})
+    set(BUILD_TYPE_SPIRV_TRANSLATOR ${BUILD_TYPE})
+    set(BUILD_TYPE_VC_INTRINSICS ${BUILD_TYPE})
+    set(BUILD_TYPE_L0 ${BUILD_TYPE})
+elseif($ENV{BUILD_TYPE})
```

Comment:
I don't really like the idea of setting `BUILD_TYPE-*` from env vars values but if you need them it is OK. At least, add a message that says that the value was derived from environment.

Reply:
Don't need them, but wasn't sure if they might be useful for passing down configurations from CI tasks. I'll add a message.

Reply:
`JOBS` in `BUILD_TYPE_*` message look like copy-paste leftover.